File: Java/UberBlack.java
package Java;

import java.util.ArrayList;
import java.util.Map;

public class UberBlack extends UberCar {
    Map<String,Map<String,Integer>> typeCarAccepted;
    ArrayList<String> seatsMaterial;

    public UberBlack(String license, Account driver, 
    Map<String,Map<String,Integer>> typeCarAccepted, ArrayList<String> seatsMaterial){
        //Use super to pass parameter to subclass (Using Inheritance)
        super(license, driver);
        this.typeCarAccepted = typeCarAccepted;
        this.seatsMaterial = seatsMaterial;
    }
}

File: Python/uberBlack.py
from uberCar import UberCar

class UberBlack(UberCar):
    #This way to use Inheritance
    typeCarAccepted = []
    seatsMaterial = []

    def __init__(self,license,driver,typeCarAccepted,seatsMaterial):
        #Super class to inheritance. Use super()
        super().__init__(license,driver)
        self.typeCarAccepted = typeCarAccepted
        self.seatsMaterial = seatsMaterial